qalab/qalab.py

```diff
@@ -83,7 +83,7 @@ def handle_command_selenium(args, logger):
                 logger.info("Selenium JAR ready at: {}".format(jar_path))
             else:
                 logger.info("Downloading selenium from : {}".format(selenium_url))
-                selenium_file = wget.download(selenium_url, out="qalab/drivers")
+                wget.download(selenium_url, out="qalab/drivers")
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the unused local variable "selenium_file". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=python%3AS1481)

Reply:
true